bigquery/google/cloud/bigquery/job.py

```diff
@@ -640,6 +641,27 @@ def schema(self, value):
             raise ValueError('Schema items must be fields')
         self._schema = tuple(value)
 
+    @property
+    def destination_encryption_configuration(self):
+        """google.cloud.bigquery.table.EncryptionConfiguration: Custom
+        encryption configuration for the destination table.
+
+        Custom encryption configuration (e.g., Cloud KMS keys) or ``None``
+        if using default encryption.
+
+        See
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load.destinationEncryptionConfiguration
+        """
+        prop = self._properties.get('destinationEncryptionConfiguration')
+        if prop is not None:
+            prop = EncryptionConfiguration.from_api_repr(prop)
+        return prop
+
+    @destination_encryption_configuration.setter
+    def destination_encryption_configuration(self, value):
+        self._properties[
```

Comment:
Is it not possible to set this to `None`?

Reply:
You're right, this needs to be fixed.

Reply:
Done.